core/unit_test/CMakeLists.txt

```diff
@@ -314,6 +314,7 @@ KOKKOS_ADD_TEST_EXECUTABLE(
 )
 # We need -rdynamic on GNU platforms for the stacktrace functionality
 # to work correctly with shared libraries
+if(NOT KOKKOS_HAS_TRILINOS)
 SET_PROPERTY(TARGET StackTraceTestExec PROPERTY ENABLE_EXPORTS 1)
```

Comment:
The issue is the target `StackTraceTestExec` is called something else with TriBITS.

Reply:
We can properly fix it later. For now I just want to get the release out. This seemed like the fastest way of getting Trilinos to pass.